[2021-05-09 19:23:39,041] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-07T00:00:00+00:00 [queued]>
[2021-05-09 19:23:39,061] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-07T00:00:00+00:00 [queued]>
[2021-05-09 19:23:39,064] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 19:23:39,068] {taskinstance.py:1069} INFO - Starting attempt 10 of 12
[2021-05-09 19:23:39,069] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 19:23:39,234] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): python_push_into_target> on 2021-05-07T00:00:00+00:00
[2021-05-09 19:23:39,238] {standard_task_runner.py:52} INFO - Started process 21627 to run task
[2021-05-09 19:23:39,245] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_4', 'python_push_into_target', '2021-05-07T00:00:00+00:00', '--job-id', '145', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/dags/dag__4.py', '--cfg-path', '/tmp/tmp5wmtahwr', '--error-file', '/tmp/tmp8nby_4hz']
[2021-05-09 19:23:39,247] {standard_task_runner.py:77} INFO - Job 145: Subtask python_push_into_target
[2021-05-09 19:23:39,617] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_4.python_push_into_target 2021-05-07T00:00:00+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-09 19:23:39,677] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=navneet@metaorigins.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag_4
AIRFLOW_CTX_TASK_ID=python_push_into_target
AIRFLOW_CTX_EXECUTION_DATE=2021-05-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-07T00:00:00+00:00
[2021-05-09 19:23:41,477] {logging_mixin.py:104} INFO - push_test
[2021-05-09 19:23:41,477] {python.py:118} INFO - Done. Returned value was: None
[2021-05-09 19:23:41,484] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=dag_4, task_id=python_push_into_target, execution_date=20210507T000000, start_date=20210509T135339, end_date=20210509T135341
[2021-05-09 19:23:41,642] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-09 19:23:41,667] {local_task_job.py:146} INFO - Task exited with return code 0
